GitTfs/Core/DerivedGitTfsRemote.cs

```diff
@@ -190,6 +190,12 @@ public string Prefix
         public bool ExportMetadatas { get; set; }
         public Dictionary<string, string> ExportWorkitemsMapping { get; set; }
 
+        public int? InitialChangeset
```

Comment:
Ok, so this property is here as a result of adding a property to the `IGitTfsRemote` interface. Per my comment on that file, please change the new property to be a method pair instead and document under what conditions each method may throw a `DerivedRemoteException` exception.